src/main/scala/cromiam/webservice/CromIamApiService.scala

```diff
@@ -44,6 +53,50 @@ trait CromIamApiService extends Directives with SprayJsonSupport with DefaultJso
     Http().singleRequest(cromwellRequest)
   }
 
+  private[webservice] def authorizeThenForwardToCromwell(user: Option[String], workflowId: String, action: String, request: HttpRequest): Future[HttpResponse] = {
+    (for {
+      _ <- completesOnceUploadComplete(request)
+      _ <- requestAuth(WorkflowAuthorizationRequest(user.getOrElse("anon"), workflowId, action))
```

Comment:
Yeah this is temporary until I can get a real test environment injecting those headers. 
FWIW you can already remove 'anon' from the allowed users in the config file